app/src/test/java/com/codesoom/assignment/application/UserServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@DisplayName("UserService 는")
+class UserServiceTest {
+    @Autowired
+    UserService userService;
+    @Autowired
+    UserRepository userRepository;
+    private final User userWithoutId = new User(null, "김갑생", "gabseng@naver.com", "12341234");
+
+    @BeforeEach
+    void setUp() {
+        userRepository.deleteAll();
+    }
+
+    @Nested
+    @DisplayName("create() 메서드는")
+    class Context_create_method {
+        @Nested
+        @DisplayName("id가 없는 User 객체를 받았을 때")
```

Comment:
"없는"은 애매한 표현입니다. 공백 문자열을 '없는'이라고 표현하는 사람도 있죠. 22번 라인을 보면 id 에 null을 입력하고 있습니다. 
```suggestion
        @DisplayName("id가 null인 User 객체를 받았을 때")
```

Reply:
그런 것 같네요! null인 구분해야겠습니다.